client/src/api/instance.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: 'http://127.0.0.1:8000/api',
```

Comment:
Use `.env` file for the REACT_APP_SERVER_API
and put authorization inside in instance
```js
const instance= axios.create({
  baseURL: `${process.env.REACT_APP_SERVER_API}/api`,
  headers: {
    'Content-Type': 'application/json',
    'Authorization': `Bearer ${TOKEN}`,
  }
});
```

Reply:
Done sir. It's working na.